### PR TITLE
Update latest-presubmits.yaml to add more trigger for lightning

### PR DESCRIPTION
**Pull request by D3Hunter**

### CODE REVIEW

**Comment by wuhuizuo** [comment]
/cc @zanmato1984 @fixdb

PTAL

**Review by fixdb — APPROVED** [review]
+1

File: prow-jobs/pingcap/tidb/latest-presubmits.yaml [hunk 1]
```diff
@@ -114,7 +114,7 @@ presubmits:
       agent: jenkins
       decorate: false # need add this.
       context: pull-lightning-integration-test
-      run_if_changed: "(lightning|pkg/domain|pkg/statistics|br/pkg/(checksum|httputil|membuf|pdutil|restore/split|storage))/.*"
+      run_if_changed: "(lightning|pkg/(domain|statistics|executor|ddl|infoschema|planner)|br/pkg/(checksum|httputil|membuf|pdutil|restore/split|storage))/.*"
```
Comment (D3Hunter) [inline prow-jobs/pingcap/tidb/latest-presubmits.yaml:117]:
```suggestion
      run_if_changed: "(lightning|pkg/(domain|statistics|ddl|infoschema)|br/pkg/(checksum|httputil|membuf|pdutil|restore/split|storage))/.*"
```

**Review by wuhuizuo — APPROVED** [review]
/lgtm
/approve

**Pull request merged**